drivers/storage/portworx/portworx.go

```diff
@@ -847,13 +849,13 @@ func (p *portworx) storageNodeToCloudSpec(storageNodes []*corev1.StorageNode, cl
 }
 
 // SetPortworxDefaults populates default storage cluster spec values
-func SetPortworxDefaults(toUpdate *corev1.StorageCluster, k8sVersion *version.Version) {
+func SetPortworxDefaults(toUpdate *corev1.StorageCluster, k8sVersion *version.Version) error {
```

Comment:
nit: currently we only init p.k8sVersion once right? I don't know if we can get more benefit refreshing p.k8sVersion every reconcile loop in `SetDefaultsOnStorageCluster`, then we can pass the correct k8s version in and don't have to change this signature here. But this change looks good to me.

Reply:
right now Init is only once, once csi is set to non-nil, there is no need to refresh k8s version anymore.